VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/participant-waiting-room.component.html

```diff
@@ -51,26 +52,31 @@ <h1 class="govuk-heading-xl">Your video hearing {{ getConferenceStatusText() }}<
     <div
       *ngIf="!showVideo && currentTime && ( hearing.isNotStarted() || hearing.isPaused() || hearing.isSuspended() || hearing.isClosed()  )"
       class="govuk-panel"
-      [ngClass]="{'hearing-on-time': hearing.isOnTime() || hearing.isPaused() || hearing.isClosed(), 'hearing-near-start': hearing.isStarting(), 'hearing-delayed': hearing.isDelayed() || hearing.isSuspended()}">
+      [ngClass]="{'hearing-on-time': hearing.isOnTime() || hearing.isPaused() || hearing.isClosed() || hearing.isStarting() || hearing.isDelayed() || hearing.isSuspended()}">
+
       <div class="govuk-body govuk-!-margin-bottom-0"
-        [ngClass]="{'hearing-on-time': hearing.isOnTime() || hearing.isPaused() || hearing.isClosed(), 'hearing-near-start': hearing.isStarting(), 'hearing-delayed': hearing.isDelayed() || hearing.isSuspended()}">
+        [ngClass]="{'hearing-on-time': hearing.isOnTime() || hearing.isPaused() || hearing.isClosed() || hearing.isStarting() || hearing.isDelayed() || hearing.isSuspended() }">
 
         <div *ngIf="hearing.isOnTime()">
           <h3 class="govuk-heading-m white">Please keep an eye on the time</h3>
           <p>Make sure you're ready when the judge starts the hearing.</p>
         </div>
 
         <div *ngIf="hearing.isStarting()">
-          <h3 class="govuk-heading-m white">Please stay near your screen</h3>
+          <!-- <h3 class="govuk-heading-m white">Please stay near your screen</h3>
+          <p>Make sure you're ready when the judge starts the hearing.</p> -->
+          <h3 class="govuk-heading-m white">Please keep an eye on the time</h3>
           <p>Make sure you're ready when the judge starts the hearing.</p>
         </div>
 
         <div *ngIf="hearing.isDelayed()">
-          <h3 class="govuk-heading-m">We're really sorry your hearing is delayed</h3>
+          <!-- <h3 class="govuk-heading-m">We're really sorry your hearing is delayed</h3>
```

Comment:
remove commented out code here if not needed please.

Reply:
this is a temp change, will not be going into master. the commented code will be reverted. its for the DAC testing for the week beginning 2nd Sept.